apps/mobile/src/features/Assets/components/NFTs/NFTs.container.tsx

```diff
@@ -60,6 +60,7 @@ export function NFTsContainer() {
       renderItem={NFTItem}
       ListFooterComponent={isFetching ? <Loader size={24} /> : undefined}
       keyExtractor={(item) => item.id}
+      style={{ marginTop: 8 }}
```

Comment:
just for the protocol, you can also use getTokenValue https://tamagui.dev/docs/core/exports#gettokenvalue

Reply:
I think we could also simply use `'$2'`